packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -146,6 +194,21 @@ export class SendConfirmation extends React.Component<Props, State> {
     )
   }
 
+  onKeyDown = (e: NativeSyntheticEvent<TextInputKeyPressEventData>) => {
```

Comment:
Would be nice if this kind of logic was baked into the textInput component. Can we add it there instead?

Reply:
Good point. Our custom `TextInput` is already pretty loaded so I've opted to create a new `CommentTextInput` component